gems/aws-sdk-cloudwatch/sig/aws-sdk-cloudwatch/auth.rbs

```diff
@@ -11,6 +11,7 @@ module AWS::SDK::CloudWatch
   module Auth
     class Params < ::Struct[untyped]
       attr_accessor operation_name (): ::Symbol
+      attr_accessor region (): untyped
```

Comment:
String

Reply:
I'll start a separate smithy-ruby PR to fix this: https://github.com/smithy-lang/smithy-ruby/blob/main/codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/AuthGenerator.java#L121

Reply:
Fixed in: https://github.com/smithy-lang/smithy-ruby/pull/183